app/views/reports/elements/_step_checklist_element.html.erb

```diff
@@ -8,7 +8,10 @@
           <span class="glyphicon glyphicon-list"></span>
       </div>
       <div class="pull-left checklist-name">
-        <em><%=t 'projects.reports.elements.step_checklist.checklist_name', name: checklist.name %></em>
+        <%= auto_link(smart_annotation_parser(simple_format(checklist.name)),
+                      link: :urls,
+                      sanitize: false,
+                      html: { target: '_blank' }).html_safe %>
```

Comment:
You didn't use locale 'projects.reports.elements.step_checklist.checklist_name'.